addons/account/models/chart_template.py

```diff
@@ -127,6 +127,7 @@ class AccountChartTemplate(models.Model):
     property_tax_payable_account_id = fields.Many2one('account.account.template', string="Tax current account (payable)")
     property_tax_receivable_account_id = fields.Many2one('account.account.template', string="Tax current account (receivable)")
     property_advance_tax_payment_account_id = fields.Many2one('account.account.template', string="Advance tax payment account")
+    property_pos_receivable_account_id = fields.Many2one('account.account.template', string="PoS receivable account")
```

Comment:
this is won't be a property field, so name it just 'default_pos_receivable_account_id' and put it before the properties